config.yml

```diff
@@ -496,12 +496,20 @@ nodes:
       - name: flags
         type: flags
         kind: ArrayNodeFlags
+        comment: |
+          Represents whether or not there is a splat operator in the array.
       - name: elements
         type: node[]
+        comment: |
+          Represent list of elements in the array.
       - name: opening_loc
         type: location?
+        comment: |
+          Represents optional source location for opening token.
```

Comment:
So this comment is effectively tautological; `opening_loc, location?` conveys the same information as `Represents optional source location for opening token.` We want to answer some questions to make this a more descriptive comment:

* Why is it optional? When is it present and when is it not?
* What tokens can be in this slot? Is it always a `[` or are there other kinds?
* How does this location correspond with the closing location?